feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-form/checkout-payment-form.component.ts

```diff
@@ -115,7 +117,8 @@ export class CheckoutPaymentFormComponent implements OnInit {
     protected globalMessageService: GlobalMessageService,
     protected fb: UntypedFormBuilder,
     protected userAddressService: UserAddressService,
-    protected launchDialogService: LaunchDialogService
+    protected launchDialogService: LaunchDialogService,
+    protected translationService?: TranslationService
```

Comment:
An `Optional` decorator is needed here, and a comment needs to be added to the doc file.
```suggestion
    protected translationService?: TranslationService
```